antora-playbook-for-publication.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Use this Antora Playbook for publication, to build the released version branch.
+site:
+  title: Eclipse Che Documentation
+  # The url property is mandatory to generate the 404 and sitemap files.
+  url: https://www.eclipse.org/che/docs
+  start_page: che-7:overview:introduction-to-eclipse-che.adoc
+  keys:
+    google_analytics: "UA-37306001-2"
+  robots: allow
+content:
+  sources:
+    - url: ./
+      branches: 7.32.x
+      edit_url: "https://github.com/eclipse/che-docs/edit/7.31.x/{path}"
```

Comment:
7.31? that's super oooooold. and not the same as line 14, either. (?)

Reply:
```suggestion
      edit_url: "https://github.com/eclipse/che-docs/edit/7.35.x/{path}"
```